src/meson.build

```diff
@@ -0,0 +1,49 @@
+libdl = cc.find_library('dl')
+
+add_project_arguments('-pthread', language : 'c')
```

Comment:
Out of curiosity, why is libpthread handled differently from libdl here?

Reply:
Because I was based on other projects [1]. I also found something like that [2] [3].
Please, let me know if it is unnecessary/wrong.

[1] http://mesonbuild.com/Users.html#
[2] http://mesonbuild.com/howtox.html#enable-threads
[3] https://github.com/mesonbuild/meson/issues/328#issuecomment-165299484